File: insert a element in the stack.py
class Stack:
    def __init__(self):
        self.stack = []

    def push(self, item):
        self.stack.append(item)

    def pop(self):
        if not self.is_empty():
            return self.stack.pop()

    def peek(self):
        if not self.is_empty():
            return self.stack[-1]

    def is_empty(self):
        return len(self.stack) == 0


# Create a stack instance
my_stack = Stack()

# Push elements onto the stack
my_stack.push(10)
my_stack.push(20)
my_stack.push(30)

# Insert an element into the stack
element_to_insert = 25
my_stack.push(element_to_insert)

#input taking by the user
taking_user=eval(input("Enter the inserting element : "))
my_stack.push(taking_user)

# Print the stack after insertion
print("Stack elements after insertion:")
while not my_stack.is_empty():
    print(my_stack.pop())




    

File: insert_a_element_in_the_stack.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author Rajeev kumar
 */
import java.util.Stack;
import java.util.Scanner;
public class insert_a_element_in_the_stack {
    public static void main(String[] args) {
        Scanner scane = new Scanner(System.in);
        Stack<Integer> stack = new Stack<>();

        // Push elements onto the stack
        stack.push(10);
        stack.push(20);
        stack.push(30);

        // Insert an element into the stack
        int elementToInsert = 25;
        stack.push(elementToInsert);
        
        //Input taking by the user 
        System.out.println("ENter the input for unserting : ");
        int num1 = scane.nextInt();
        stack.push(num1);

        // Print the stack after insertion
        System.out.println("Stack elements after insertion:");
        while (!stack.isEmpty()) {
            System.out.println(stack.pop());
        }
    }
}

